Userland/Libraries/LibGUI/CommonActions.cpp

```diff
@@ -17,12 +17,12 @@ namespace GUI {
 
 namespace CommonActions {
 
-NonnullRefPtr<Action> make_about_action(DeprecatedString const& app_name, Icon const& app_icon, Window* parent)
+NonnullRefPtr<Action> make_about_action(StringView app_name, Icon const& app_icon, Window* parent)
 {
     auto weak_parent = AK::make_weak_ptr_if_nonnull<Window>(parent);
     auto action = Action::create(DeprecatedString::formatted("&About {}", app_name), app_icon.bitmap_for_size(16), [=](auto&) {
         AboutDialog::show(
-            String::from_deprecated_string(app_name).release_value_but_fixme_should_propagate_errors(),
+            String::from_utf8(app_name).release_value_but_fixme_should_propagate_errors(),
```

Comment:
This makes me think we should make it a `String` all of the way down, since `AboutDialog` stores a `String` as a member anyway

Reply:
Updated to use `String` instead :^)